src/client/components/ProductsContainer.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import ProductsList from './ProductsList';
+
+class ProductsContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    const { title, products, isArchive } = this.props;
+    this.state = {
+      isArchive,
+      products,
+      title
+    };
+  }
+
+  render() {
+    const { title, products, isArchive } = this.state;
+    return (
+      <div className="products">
+        <header className="products__header">
+          <h2 className="products__heading">{title}</h2>
+          <span className="products__info">
+            {isArchive ? 'Remove item' : 'Mark as ordered'}
```

Comment:
We have two different behaviors, for the similar cases:
- ```title``` is passed to the component by props. Value of that passed props is determined in parent component by isArchive value, 
- "Remove item" and "Mark as ordered" labels depends on ```isArchive``` boolean